test/integration/testsuite/prepare.go

```diff
@@ -114,12 +114,13 @@ func setUp() string {
 		ensureTestDirEmpty()
 	}
 
-	err = os.Setenv(constants.MiniShiftHomeEnv, testDir)
+	testDefaultHome := fmt.Sprintf("%s/.minishift", testDir)
+	err = os.Setenv(constants.MiniShiftHomeEnv, testDefaultHome)
 	if err != nil {
 		fmt.Printf("Error setting up environmental variable %v: %v\n", constants.MiniShiftHomeEnv, err)
 	}
 
-	return testDir
+	return testDir, testDefaultHome
```

Comment:
@agajdosi  I am not able to recall what is the use of testDir. It is for temporarily storing the logs and artifacts, right?

Reply:
@agajdosi  with your changes should  I expect the `.minishift` inside of `/tmp/minishift-tmp-test-dir-556220581`?